test/test_utils.erl

```diff
@@ -803,7 +803,7 @@ frame_packet(MType, PubKeyBin, NwkSessionKey, AppSessionKey, FCnt, Options) ->
         type = lorawan,
         payload = Payload1,
         frequency = 923.3,
-        datarate = maps:get(datarate, Options, <<"SF8BW125">>),
+        datarate = maps:get(datarate, Options, "SF8BW125"),
```

Comment:
FYI - The Erlang LoRaWAN library assumes data rates are always of the form <<"SF8BW125">>.  It could handle the second form you changed it to, but not yet...

Reply:
protobufs always expect strings for datarate.